### DISCUSSION TITLE
How to create a table dynamically in server pod

### DISCUSSION THREAD

**Discussion by SriRam-Macha** [serverpod/serverpod#723] [Q&A] [answered]
lets say I have multiple users. They want to chat with each other, I want to create a table whenever a user starts a conversation with another user. I have a predefined format for messages which will be the rows of this table (for ex - Message(id, text, timestamp, Fromwhichuser)). How will I create the tables dynamically after serverpod is started and not running the SQL script manually to add the tables?

**Comment by AvoMandjian** [comment]
The ability to create tables and generate code programmatically is also an important feature when you are creating a CMS with serverpod

**Comment by vlidholt** [comment, marked as answer]
You can use custom SQL queries to create tables. This is not supported through the Serverpod ORM.

However, for the use case you are describing, it's sounds like it would be better to create a structure where you just add rows for your conversations.
